File: c files/Student Marks.c
#include <stdio.h>

int main() {
    int n;
    printf("Enter number of students\n");
    scanf("%d", &n);

    char name[n][50];
    int mark1[n], mark2[n], mark3[n], rollno[n], total[n];
    float average[n];

    for (int i = 0; i < n; i++) {
        printf("Enter the name of student %d: ", i + 1);
        scanf("%s", name[i]);
        printf("Enter roll number of student %d: ", i + 1);
        scanf("%d", &rollno[i]);
        printf("Enter mark1: ");
        scanf("%d", &mark1[i]);
        printf("Enter mark2: ");
        scanf("%d", &mark2[i]);
        printf("Enter mark3: ");
        scanf("%d", &mark3[i]);

        total[i] = mark1[i] + mark2[i] + mark3[i];
        printf("Total marks are %d\n", total[i]);

        average[i] = (float) total[i] / 3;
        printf("Average marks is %.2f\n", average[i]);
    }

    return 0;
}

File: py files/Student Marks.py
print("Enter number of students")
n = int(input())
name = [""] * (n)
mark1 = [0] * (n)
mark2 = [0] * (n)
mark3 = [0] * (n)
rollno = [0] * (n)
total = [0] * (n)
average = [0] * (n)

for i in range(0, n - 1 + 1, 1):
    print("Enter the name of student" + str(i + 1))
    name[i] = input()
    print("Enter roll number of student" + str(i + 1))
    rollno[i] = int(input())
    print("Enter mark1")
    mark1[i] = int(input())
    print("Enter mark2")
    mark2[i] = int(input())
    print("Enter mark3")
    mark3[i] = int(input())
    total[i] = mark1[i] + mark2[i] + mark3[i]
    print("Total marks are  " + str(total[i]))
    average[i] = float(total[i]) / 3
    print("Average marks is  " + str(average[i]))
